checker/src/main/java/org/checkerframework/checker/nullness/NullnessVisitor.java

```diff
@@ -380,6 +381,20 @@ public Void visitIf(IfTree node, Void p) {
         return super.visitIf(node, p);
     }
 
+    @Override
+    public Void visitInstanceOf(InstanceOfTree node, Void p) {
+        if (node.getType().getKind() == Kind.ANNOTATED_TYPE) {
+            AnnotatedTypeMirror type = atypeFactory.fromExpression((ExpressionTree) node.getType());
```

Comment:
Could you:
- cast to `AnnotatedTypeTree`
- call `TreeUtils.annotationsFromTree()`
- use `AnnotationUtils.containsSame` to look for the annotations

Then you don't need to change the visibility below.

Reply:
Thanks for the help! I have used the above methods and reset the visibility of ```fromExpression``` to ```private``` in AnnotatedTypeFactory.